pkg/flare/archive.go

```diff
@@ -187,7 +187,7 @@ func zipConfigFiles(zipFile *archivex.ZipFile, hostname string, confSearchPaths
 			if err != nil {
 				return e
 			}
-			fileName := filepath.Join(hostname, "etc", "datadog.yaml")
+			fileName := filepath.Join(hostname, "etc", "runtime_config_dump.yaml")
```

Comment:
this is the actual config file that's been used (if any), `datadog.yaml` is legit here

Reply:
Ah okay, that's what I thought; I was told otherwise. I'll revert it now.